README.md

```diff
@@ -91,11 +91,33 @@ Within the payload, you can declare the user's roles as an array. Currently the
 {
   "iat": 1516239022,
   "exp": 1516298489,
-  "roles": ["moderator"]
+  "roles": ["moderator"],
 }
 ```
+## Board name verification in the JWT
 
-Moderators have access to the Clear tool, which will wipe all content from the board.
+WBO supports verification of the board with a JWT. The board name should be passed in the JWT in the `room_name` claim in the JWT.
+
+The `AUTH_SECRET_KEY` variable in [`configuration.js`](./server/configuration.js) should be filled with the secret key for the JWT.
+The `CHECK_BOARDNAME_IN_JWT` variable in [`configuration.js`](./server/configuration.js) should be set to `true`
+
+```
+{
+  ....
+  "board_name": "<boardName>"
```

Comment:
This is not a standard property. Can we reuse the existing roles property ?

```json
{ 
   "roles": ["editor:my_first_board", "editor:my_second_board"],
}
```

?